### PR TITLE
Add location property to BigQuery data source settings

### PR DESCRIPTION
**Pull request by kyoshidajp**
# Feature

BigQuery began to support Tokyo region from April this year.

- [Google Cloud Platform Blog: BigQuery arrives in the Tokyo region](https://cloudplatform.googleblog.com/2018/04/BigQuery-arrives-in-the-Tokyo-region.html)
- [Dataset Locations  \|  BigQuery  \|  Google Cloud](https://cloud.google.com/bigquery/docs/dataset-locations)

This PR adds a location property to BigQuery data source settings. And calls job query with it.

If it is not specified, the value is `US` (which is default BigQuery location).

# Screenshot

bq_location

### CODE REVIEW

**Comment by sekiyama58** [comment]
Thanks kyoshidajp!
@arikfr Could you merge this one?

File: redash/query_runner/big_query.py [hunk 1]
```diff
@@ -148,10 +155,15 @@ def _get_bigquery_service(self):
     def _get_project_id(self):
         return self.configuration["projectId"]
 
+    def _get_location(self):
+        location = self.configuration.get("location", "")
+        return "US" if location == "" else location
```
Comment (arikfr) [inline redash/query_runner/big_query.py]: I think you can simplify this into:

```python
def _get_location(self):
    return self.configuration.get("location", "US")
```

No?

File: redash/query_runner/big_query.py [hunk 2]
```diff
@@ -110,6 +113,10 @@ def configuration_schema(cls):
                     "type": "boolean",
                     'title': "Use Standard SQL (Beta)",
                 },
+                'location': {
+                    "type": "string",
+                    'title': 'Processing Location (default: US)',
```
Comment (arikfr) [inline redash/query_runner/big_query.py]: For default values we have the `default` property of the configuration item.

**Comment by kyoshidajp** [comment]
@arikfr Thanks. I fixed. Could you review again?

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**